mbuild/compound.py

```diff
@@ -126,6 +126,20 @@ def load(filename_or_object, relative_to_module=None, compound=None, coords_only
             compound = read_xyz(filename_or_object, compound=compound)
         return compound
 
+    if extension == '.sdf':
+        pybel = import_('pybel')
+        pybel_mol = pybel.readfile('sdf', filename_or_object)
+        # pybel returns a generator, so we grab the first molecule of a list of len 1
+        # Warn user if there are more molecules
+        pybel_mol = [i for i in pybel_mol]
+        if len(pybel_mol) == 1:
+            compound.from_pybel(pybel_mol[0])
+        else:
+            compound.from_pybel(pybel_mol[0])
+            warn("More than one pybel molecule in file, more than one pybel "
+                 "molecule is not supported, using {}".format(filename_or_object))
```

Comment:
I'm confused, so it loads the data in the same way whether or not it's one or multiple molecules? If this is what we want (I'm fine with it), then it should be

```python3
        compound.from_pybel(pybel_mol[0])
        if len(pybel_mol) > 1:
            warn("More than one pybel molecule in file, more than one pybel "
                 "molecule is not supported, reading only the first molecule "
                 "of {}".format(filename_or_object))

Reply:
It seems like regardless if we read in 1 or multiple molecules, a generator of length 1 (when converted to a list) is returned.

Reply:
I'm actually not sure how to load in a SDF file with multiple pybel molecules.  I'm thinking now that the warning should be changed to an error exception.  That way, the unit tests will inform us if this behavior in Pybel changes in the future.  Any thoughts?

Reply:
Good to me